### PR TITLE
Restify PEP 206

### PR DESCRIPTION
**Pull request by Mariatta**
Straightforward conversion.
I used my (imperfect) script at https://github.com/Mariatta/restify_me for this. 
Feedback are welcome :)

### CODE REVIEW

File: pep-0206.txt [hunk 1]
```diff
@@ -5,112 +5,124 @@ Last-Modified: $Date$
...
 
 
 Acknowledgements
-        
-    The PEP is based on an earlier draft PEP by Moshe Zadka, titled
-    "2.0 Batteries Included."
+================
+
+The PEP is based on an earlier draft PEP by Moshe Zadka, titled
+"2.0 Batteries Included."
```
Comment (Mariatta) [inline pep-0206.txt]: This PEP does not have the copyright notice, something that's discussed in #123 
Should it be added here?

Comment (gvanrossum) [inline pep-0206.txt]: Maybe @akuchling (the PEP author) can add one or tell you what it should be? (I presume public domain, like all other PEPs, but it's polite to ask.)

Comment (Mariatta) [inline pep-0206.txt]: Thanks @gvanrossum :)
I won't make changes wrt copyright notice without @akuchling 's input.
If anything, perhaps it should go in a separate PR anyway?

**Comment by brettcannon** [comment]
So if we skip adding a copyright is this PR ready to be committed?

**Comment by Mariatta** [comment]
I tested it locally, and it renders fine.
So, maybe it's ok to merge?  :)

**Comment by brettcannon** [comment]
Travis is green so that's good enough for me. 😄

**Comment by brettcannon** [comment]
Thanks, BTW!
